### PR TITLE
Feat: MVP features for soft launch

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
I could use some input on the verbiage and UI, but I think we want something like this.

### CODE REVIEW

**Comment by ojeytonwilliams** [comment]
The user stories have changed a bit, so this needs updating.

File: config/certification-settings.ts
Comment (nhcarrigan) [inline config/certification-settings.ts:21]: Is this correct after #44535 lands?

Comment (ojeytonwilliams) [reply config/certification-settings.ts:21]: Good catch.  Nope, it'll need renaming.

**Comment by ojeytonwilliams** [comment]
Sorry for the rebase, @nhcarrigan, I was checking that it works as expected with `SHOW_UPCOMING_CHANGES=false`

**Review by naomi-lgbt — DISMISSED** [review]
Fixed the conflict - this is looking good to me.

**Comment by raisedadead** [comment]
Ah - sorry, should we hold off merging localization PRs for a bit? Not that we are doing any deployments for a couple of days.

**Comment by nhcarrigan** [comment]
I think we should be fine - the conflict here was relatively minor.

File: client/src/resources/cert-and-project-map.ts
Comment (ojeytonwilliams) [inline client/src/resources/cert-and-project-map.ts:733]:
```suggestion
  // TODO: for the hard launch, the conditional should just be `showNewCurriculum`
```

**Review by naomi-lgbt — APPROVED** [review]

**Review by moT01 — APPROVED** [review]

**Review by ShaunSHamilton — APPROVED** [review]
Oliver is investigating Certification links on `/settings`. Otherwise LGTM 👍

**Pull request merged**
